### PR TITLE
ESX option

### PR DESCRIPTION
**Pull request by utkuali**
btw great script!

### CODE REVIEW

File: DiamondBlackjack/sv_blackjack.lua [hunk 1]
```diff
@@ -40,6 +61,21 @@ AddEventHandler('playerDropped', function (reason)
...
+    print("===============")
+    print(dump(blackjackGameData))
+end)
+
+RegisterCommand("debugcarddata",function()
+    print("carddata")
+    print("===============")
+    print(dump(blackjackGameData[1024]))
+end)
+
```
Comment (CreepPork) [inline DiamondBlackjack/sv_blackjack.lua:78]: Why is this modified in this PR? Seems completely unrelated.

Comment (TheLaw113) [reply DiamondBlackjack/sv_blackjack.lua:78]: can you help me I tried to add your script in esx server but it says this in console 

Error parsing script @DiamondBlackjack/sv_blackjack.lua in resource DiamondBlackjack: @DiamondBlackjack/sv_blackjack.lua:34: 'end' expected (to close 'function' at line 30) near 'if'
Failed to load script sv_blackjack.lua.

**Review by TheLaw113 — CHANGES_REQUESTED** [review]
